metadata-ingestion-examples/common/build.gradle

```diff
@@ -7,7 +7,6 @@ dependencies {
 
   compile externalDependency.javaxInject
   compile externalDependency.kafkaAvroSerde
-  compile externalDependency.kafkaSerializers
```

Comment:
These build files were changed in a different pr. Remove?

Reply:
I need to rebase these in to pass CI.